jwql/website/apps/jwql/views.py

```diff
@@ -38,10 +38,12 @@
 """
 
 import csv
+import pandas as pd
```

Comment:
Since `pandas` is a third-party library, this should do in the next import section (and in alphabetical order)

Reply:
Gotcha, will make the change 👍